src/framework/dockwindow/CMakeLists.txt

```diff
@@ -81,3 +81,4 @@ set(MODULE_QML_IMPORT ${CMAKE_CURRENT_LIST_DIR}/qml)
 
 setup_module()
 
+target_no_warning(${MODULE} -WMSVC-no-unreachable) # workaround for a warning generated by Qt 5.9.1 code
```

Comment:
This doesn't seem to help, there are still 3 such warnings, in local builds as well as on GitHub CI